playbooks/roles/l2tp-ipsec/templates/instructions-fr.md.j2

```diff
@@ -0,0 +1,127 @@
+- - -
+{% for key, value in streisand_languages.iteritems() %}
+  [{{ value.language_name }}](index{{ value.file_suffix }}.html)&nbsp;
+{% endfor %}
```

Comment:
This header fragment is added in all of the .md template files for both languages. Is it possible to put it in its own `.md.j2` file and `{% include %}` it everywhere? I haven't tried it. If it's a huge pain its probably not worth it but it would be nice if it works easily!